iamclient/iamclient.go

```diff
@@ -192,15 +192,14 @@ func (client *Client) InstallCertificates(
 		response, err := client.certificateService.ApplyCert(ctx, request)
 		if err == nil {
 			certConfirmation.Status = "installed"
+			certConfirmation.Serial = response.Serial
```

Comment:
What is the problem with filling serial all the time?

Reply:
response is nil